src/api/docs/content/specs/queries.yaml

```diff
@@ -208,12 +208,10 @@ components:
                   time:
                     type: number
                     description: Time until the response was received (ms, negative if N/A)
-              ttl:
-                type: integer
-                description: Query's Time-To-Live (TTL) value (`-1` if N/A)
               regex_id:
                 type: integer
-                description: ID of blocking regex (`-1` if N/A)
+                description: ID of blocking regex (`NULL` if N/A)
```

Comment:
Does it only save the regex_id in case it's a blocking regex? Are allowing regex ignored here?